### PR TITLE
Prep for 1.2.1 announcement

### PR DESCRIPTION
**Pull request by gaktive**
May need to add Stratos 2.1.0 here if they're aligning their release with CAP.

### CODE REVIEW

File: adoc/Release-Notes.adoc [hunk 1]
```diff
@@ -21,6 +21,67 @@ endif::[]
 [id='sec.major-change']
 == Major Changes
 
+[id='sec.1_2_1']
+=== Release 1.2.1, September 2018
+
+[id='sec.1_2_1.issue']
+==== Known Issues
+
+* If you are upgrading to {product} 1.2.1, you will need to run the `helm upgrade` command with two additional parameters: `helm upgrade --force --recreate-pods`. This will be the case with all future 1.x upgrades due to the expected behaviour with Kubernetes that unready pods won’t get upgraded.
```
Comment (ghost) [inline adoc/Release-Notes.adoc]: Suggestion that puts the "issue" first, then discusses the "solution" which improves clarity and also use simpler tenses:
"Starting with {product} 1.2.1, during `helm upgrade`, Kubernetes will not upgrade pods that are not ready by default. To upgrade them anyway, use the following amended command instead: `helm upgrade --force --recreate-pods`."

I hope I have not misunderstood the content. Feel free to adjust.

File: adoc/Release-Notes.adoc [hunk 2]
```diff
@@ -21,6 +21,67 @@ endif::[]
...
+
+* Similar to {caaspa} 3, {azure} uses an updated version of Kubernetes that mandates a stricter security policy via PodSecurityPolicy (PSP), which is included as part the {product} Deployment Guide. Any namespace tied to {product} requires privileged ports to be accessible needs to have to have a PSP set appropriately for access. This would include the default conventions of `scf`, `uaa`, `stratos-ui`, `mysql-sidecar` and `postgres-sidecar` as per our documentation.
```
Comment (ghost) [inline adoc/Release-Notes.adoc]: `as per our documentation`... -> Can you add a link to a specific section instead?

File: adoc/Release-Notes.adoc [hunk 3]
```diff
@@ -21,6 +21,67 @@ endif::[]
...
+
+* {azure} users will need to add `--node-osdisk-size=60` as part of their AKS creation command to ensure the root partition has enough space for the install and future upgrades. The {product} Deployment Guide has a section called `Create Resource Group and AKS Instance` and the `node-osdisk-size` parameter would need to be added to the `az aks create` command.
```
Comment (ghost) [inline adoc/Release-Notes.adoc]: Could you add the expansion of the acronym "AKS" somewhere and again, do "1. issue, 2. solution"?

Suggestion:
"If you are using {azure}, ensure that the root partition has enough space for the installation and potential upgrades. To do so, add the parameter `--node-osdisk-size=60` to the command that creates the AKS (Azure Kubernetes Service) instance (`az aks create`). For more information about this parameter, see [insert link here]."

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Sorry for missing overall review comment, I am a fast clicker -- changes look good generally but sometimes wording is a bit complicated.

(Also the comments that GitHub shows as outdated aren't really outdated.)

File: adoc/Release-Notes.adoc (hunk 1 above)
Comment (gaktive) [reply adoc/Release-Notes.adoc]: Makes total sense. I'll review the rest of your feedback in the "outdated" comments, thanks!

File: adoc/Release-Notes.adoc (hunk 2 above)
Comment (gaktive) [reply adoc/Release-Notes.adoc]: Added link to CaaSP 3 documentation, docs ticket filed to update Azure/AKS documentation then we can consider updating link. The copy used matches what was recorded in 1.2.0.

File: adoc/Release-Notes.adoc (hunk 1 above)
Comment (gaktive) [reply adoc/Release-Notes.adoc]: Updated entry, though using some of the similar 1.2.0 copy.

File: adoc/Release-Notes.adoc (hunk 3 above)
Comment (gaktive) [reply adoc/Release-Notes.adoc]: AKS expansion added and link to relevant section in deployment guide.

**Review by ghost — APPROVED** [review]

**Comment by thardeck** [comment]
lgtm

**Comment by troytop** [comment]
Reviewed. Looks good! No edits.

**Pull request merged**